docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+To illustrate the usage of the audit scanner in Kubewarden, let's consider the
+following scenario:
+
+Assume you have deployed a set of policies to enforce specific rules and
+validations on your Kubernetes cluster. However, some resources were deployed
+before these policies were implemented, and you want to ensure that these
+resources comply with the newly deployed policies.
+
+To address this need, Kubewarden introduced the audit scanner component
+starting from version `v1.7.0`. The audit scanner periodically evaluates the
+resources running in your cluster and checks if they comply with the policies
+that have been deployed. By doing so, it helps operators identify any resources
+that violate the validations performed by the policies, even if they were
+deployed before the policies themselves.
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner
+
```

Comment:
I think we should add an `### Intrinsic limitations`  section. Copy-pasted and trimmed from the RFC:

```
## Audit intrinsic limitations

### Policies checking `UPDATE` events

Policies can inspect `CREATE`, `UPDATE`, and `DELETE` events. The audit check
feature cannot intrinsically test for `UPDATE` events, as it doesn't know
exactly which part of the resource needs to `UPDATE` in a meaningful way.

Policies interested in `UPDATE` events will be run by the audit check, and
exercised for other operations, but not the `UPDATE` operation.

### Policies checking user and user group information

Each Kubernetes admission request object contains information about which user
(or ServiceAccount) initiated the event, and to which group they belong.

The audit feature cannot possibly infer what user or ServiceAccount to test
against, and will not produce a meaningful evaluation, as it will use its own
ServiceAccount.

The policy author should opt out of audit checks by setting
`spec.BackgroundAudit` to `false` for the policy.

### Policies relying on external data

Policies can request and use external data when performing an evaluation. These
policies can be evaluated by the audit checks, but their outcome can change over
time depending on the external data.

### Policies targetting `*`

Policies targetting any kind of Kubernetes resources are not being evaluated for
now. Users have to be explicit about the Kubernetes resources a policy targets
for now.

```